src/utils/storage-utils.js

```diff
@@ -13,6 +13,18 @@ import {
 
 const rasterImageFileTypes = ['image/png', 'image/jpg', 'image/jpeg', 'image/webp'];
 
+export function getMediaLibraryPath() {
+  return 'media-library/';
```

Comment:
I know we use `urlUtils.concatParts` in the place where this is called to append the file name, but as the other two helpers `getDocumentMediaDocumentPath` and `getRoomMediaRoomPath` do not append a slash at the end, I wonder why you added a trailing slash in this one?

Reply:
No need to keep wondering, removed! 😅